web/src/main/java/org/akaza/openclinica/control/admin/CreateXformCRFVersionServlet.java

```diff
@@ -273,16 +281,11 @@ private XformContainer parseInstance(String xform, Errors errors, Html html, Str
                 }
             }
 
-            XPathFactory xPathfactory = XPathFactory.newInstance();
-            XPath xpath = xPathfactory.newXPath();
-            XPathExpression expr = xpath.compile("/html/body");
-
             // List<Body> body = (List<Body>) html.getBody();
```

Comment:
Probably from before but feel free to remove commented code for better readability